common/app/model/meta.scala

```diff
@@ -211,13 +217,23 @@ object MetaData {
   }
 }
 
+final case class ContentFormat(
```

Comment:
Minor, but `Format` (with the `Content` prefix) might be okay here as `Format` is the term used in our other services and all of these fields relate to content so that bit feels redundant.

Reply:
Need to use `ContentFormat` for the sake of ease here. `Format` is used for the play JSON api and it's a nightmare going through and working out all the places it's used. This is made worse by the fact that it fails on TeamCity because of a namespace clash, but not when I compile and run locally. So using `ContentFormat` is the best approach given how often `Format` is imported as part of `import play.api.libs.json._`